### PR TITLE
Manual python3.13

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

**Comment by bollwyvl** [comment]
@conda-forge/uvloop ready for review :loop:

**Review by mariusvniekerk — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks Nick! 🙏

So normally the strategy of recythonizing makes sense

Unfortunately we may still not have enough Python 3.13 support in Cython itself: https://github.com/cython/cython/issues/6368

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,39 +1,49 @@
...
+  patches:
+    - 0000-always-cython.patch
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . -vv --global-option='build_ext' --global-option='--cython-always' --global-option='--use-system-libuv'
+  number: 1
   skip: true  # [py<38 or win]
+  script:
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```
Comment (jakirkham) [inline recipe/meta.yaml]: Is it possible to skip the patch and use the option flags here? This seemed to work well

Patching can sometimes be finicky with conflicts and such

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,39 +1,49 @@
...
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython <3                              # [build_platform != target_platform]
+    - cython >=3,<4                          # [build_platform != target_platform]
+    - python                                 # [build_platform != target_platform]
+    - setuptools >=60                        # [build_platform != target_platform]
+    - wheel                                  # [build_platform != target_platform]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Am curious why `setuptools` and `wheel` are now needed in `requirements/build`

Can understand `requirements/host`

File: recipe/meta.yaml [hunk 3]
```diff
@@ -46,7 +56,6 @@ about:
     uvloop is a fast, drop-in replacement of the built-in asyncio event loop.
     uvloop is implemented in Cython and uses libuv under the hood.
   doc_url: http://uvloop.readthedocs.org
-  dev_url: https://github.com/MagicStack/uvloop
```
Comment (jakirkham) [inline recipe/meta.yaml:49]: Know this is redundant, but let's leave it. Some folks like having this stuff show up in the README and Anaconda.org

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml (hunk 3 above)
Comment (bollwyvl) [reply recipe/meta.yaml:49]: 94a1f7e

File: recipe/meta.yaml (hunk 2 above)
Comment (bollwyvl) [reply recipe/meta.yaml]: Tried in 94a1f7e, will see if it works

File: recipe/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipe/meta.yaml]: 94a1f7e

**Comment by bollwyvl** [comment]
> not have enough Python 3.13 support in Cython

Yeah, dunno: https://pypi.org/project/uvloop/0.21.0/#files shows some py313 wheels with `.pyx` and `.pxd` in them, so guess they worked around whatever limitations are present.

**Comment by bollwyvl** [comment]
@conda-forge/uvloop ready for review

**Comment by bollwyvl** [comment]
Doing the morning rounds: @conda-forge/uvloop anything else missing here? Would like to get this through to start looking at verifying python3.13 in `anyio`, `uvicorn`, etc.

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

**Comment by bollwyvl** [comment]
:bellhop_bell: @conda-forge/uvloop (now that CI is running again)

**Comment by bollwyvl** [comment]
Thank you!

**Pull request merged**